### DISCUSSION TITLE
Table Column Resizing

### DISCUSSION THREAD

**Discussion by Dwindle2234** [TriliumNext/Trilium#3887] [General]
Is it just me that can't resize a table's column widths?  It looks so easy on the CKEditor page (https://ckeditor.com/docs/ckeditor5/latest/features/tables/tables.html) but they never 'highlight' or become selectable in Trilium (neither Firefox or Chrome)?

Any help appreciated.

**Comment by sigaloid** [comment]
Related issue: https://github.com/zadam/trilium/issues/3875
